rpcs3/Emu/Io/Null/NullPadHandler.h

```diff
@@ -64,7 +64,7 @@ class NullPadHandler final : public PadHandlerBase
 		return true;
 	}
 
-	void ThreadProc() override
+	void process() override
```

Comment:
Maybe call it "run" or operator ()
Usually proc in the context of an executor (such as threadproc) means procedure and not process.

Reply:
It's not a thread though. It's basically just one snapshot of the current state of each handler

Reply:
:ok_hand:

Reply:
thing->process sounds weird, but its just a preference at the end of the day.